types/url-parse/index.d.ts

```diff
@@ -43,6 +46,7 @@ interface URLParse {
 declare const URLParse: {
     new(address: string, location?: string | object, parser?: boolean | URLParse.QueryParser): URLParse;
     (address: string, location?: string | object, parser?: boolean | URLParse.QueryParser): URLParse;
+    (address: string, parser?: boolean | URLParse.QueryParser): URLParse;
```

Comment:
Please change the order of overloads with overload with fewer parameters coming first. Thanks